src/System.Buffers.ReaderWriter/System/Buffers/Reader/BufferReader_search.cs

```diff
@@ -321,23 +321,27 @@ public bool TryReadTo(out ReadOnlySequence<T> sequence, T delimiter, T delimiter
         /// <param name="delimiters">The delimiters to look for.</param>
         /// <param name="advancePastDelimiter">True to move past the first found instance of any of the given <paramref name="delimiters"/>.</param>
         /// <returns>True if any of the the <paramref name="delimiters"/> were found.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Heh, whoops. Left this in. I was intending to pull this, but I'll leave it for now to illustrate what I was talking about.